crates/noirc_evaluator/src/ssa/opt/mem2reg.rs

```diff
@@ -804,6 +804,7 @@ mod tests {
 
         let main = ssa.main();
         assert_eq!(main.reachable_blocks().len(), 2);
+        println!("{ssa}");
```

Comment:
```suggestion
```
We want these prints or did you add them for testing?

Reply:
They were added for testing but only show if the tests fail. I'll just remove them since they're not necessary.